src/sentry/static/sentry/app/chartcuterieConfig.tsx

```diff
@@ -0,0 +1,103 @@
+/* global process */
+
+/**
+ * This module is used to define the look and feels for charts rendered via the
+ * backend chart rendering service Chartcuterie.
+ *
+ * Be careful what you import into this file, as it will end up being bundled
+ * into the configuration file loaded by the service.
+ */
+
+import type {
+  ChartcuterieConfig,
+  RenderConfig,
+  RenderDescriptor,
+} from '@sentry/chartcuterie/lib/types';
+
+import Grid from 'app/components/charts/components/grid';
+import Legend from 'app/components/charts/components/legend';
+import XAxis from 'app/components/charts/components/xAxis';
+import YAxis from 'app/components/charts/components/yAxis';
+import AreaSeries from 'app/components/charts/series/areaSeries';
+import {getColorPalette} from 'app/components/charts/utils';
+import {EventsStatsData} from 'app/types';
+import {lightTheme as theme} from 'app/utils/theme';
```

Comment:
I believe that we should avoid renaming module imports

Reply:
I think in this case it's OK because of the context and ease-of-use, since these objects expect `theme` keys.